src/FunctionApps/python/GenerateCSVs/README.md

```diff
@@ -10,3 +10,19 @@ This function app requires config to be present. Locally these are settings in `
     * There must be a sub-structure of record types (all caps) under the prefix (eg: `[CSV_INPUT_PREFIX]`VXU, `[CSV_INPUT_PREFIX]`ELR)
 * `CSV_OUTPUT_PREFIX`: a string prefix where FHIR CSV output files are stored within the container
     * Do not include a trailing /.
+
+## Running via CLI
+
+Run `generate_csvs.py` from the python function root via something like:
+
+```
+python generate_csvs.py "https://pitestdatasa.blob.core.windows.net/bronze" "blob-trigger-out/valid-messages/" vxu > /tmp/vxu.csv"
```

Comment:
Does this mean the script version doesn't iterate over the given directory for all three filetypes anymore?

Reply:
Or rather, it looks like all of the three data types are being combined into a single file, since there aren't three output files anymore? Not clear on how this is happening but I would definitely like this tool to be able to generate three files, since we'll want to combine them or not for depending on our use case (e.g. for May 3 we may just be looking at ECR + ELR equity stuff).

Reply:
Looks like this will do just one record type at a time, so we would need to run generate_csvs.py 3 times to make all 3 csvs. Given the current situation this seems fine to me.

Reply:
it's one record type at a time - figured it may be helpful in case

a. we're having issues with only one type or
b. we want to try to run them in parallel